src/referentielijsten_client/tests/test_client.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+from django.core.cache import cache
+
+from maykin_common.vcr import VCRTestCase
+from zgw_consumers.models import Service
+from zgw_consumers.test.factories import ServiceFactory
+
+from referentielijsten_client.client import ReferentielijstenClient
+
+
+class ReferentielijstenClientTestCase(VCRTestCase):
+    service_slug = "referentielijsten-api"
+    VCR_TEST_FILES = Path("src/referentielijsten_client/tests")
+
+    def _get_items_from_fixture(self, tabel_code: str):
+        return self.client.get_cached_items_by_tabel_code(tabel_code)
+
+    def setUp(self):
+        super().setUp()
+        ServiceFactory(slug=self.service_slug, api_root="http://localhost:8004/api/v1")
+        self.service = Service.objects.get(slug=self.service_slug)
+        self.client = ReferentielijstenClient(
+            service=self.service, base_url=self.service.api_root
+        )
+        cache.clear()
+
+    def test_get_items_by_tabel_code(self):
+        items = self.client.get_items_by_tabel_code("KANAAL")
+
+        self.assertIsInstance(items, list)
+        self.assertTrue(all("code" in item for item in items))
+        codes = [item["code"] for item in items]
+        self.assertIn("email", codes)
+        self.assertIn("phone", codes)
+
+    def test_get_cached_items_by_tabel_code(self):
+        first = self._get_items_from_fixture("KANAAL")
+        second = self._get_items_from_fixture("KANAAL")
```

Comment:
It would be nice if this test also verifies that only one network call happens (because it's cached by the underlying method). I don't fully remember how to check this, but I think maybe with something like `self.vcr._current_cassette.requests` you could check this